include/lsst/meas/modelfit/CModel.h

```diff
@@ -305,12 +301,15 @@ struct CModelControl {
 
     CModelControl() :
         psfName("DoubleGaussian"),
-        minInitialRadius(0.1)
+        minInitialRadius(3E-3),
+        fallbackInitialMomentsPsfFactor(1.5)
     {
         initial.nComponents = 3; // use very rough model in initial fit
         initial.optimizer.gradientThreshold = 1E-2; // with coarse convergence criteria
         initial.optimizer.minTrustRadiusThreshold = 1E-2;
         dev.profileName = "luv";
+        exp.nComponents = 6;
+        exp.optimizer.maxOuterIterations = 250;
     }
```

Comment:
The changes in CModelControl appear to come from HSC-1264. This ticket also changed some default values in meas_extensions_multiShapelet, which has been removed from LSST, as part of commit aaf9922f2963a4a6750f60552742eaaa16839f72.  Please verify that the changes to the default values in multiShapelet are appropriately set in the new LSST location, or are not needed in the LSST stack